src/config/config_type.rs

```diff
@@ -305,12 +305,12 @@ macro_rules! create_config {
             ///
             /// Return a `Config` if the config could be read and parsed from
             /// the file, Error otherwise.
-            pub(super) fn from_toml_path(file_path: &Path) -> Result<Config, Error> {
+            pub(super) fn from_toml_path(file_path: &Path) -> Result<Config, ::failure::Error> {
                 let mut file = File::open(&file_path)?;
                 let mut toml = String::new();
                 file.read_to_string(&mut toml)?;
                 Config::from_toml(&toml, file_path.parent().unwrap())
-                    .map_err(|err| Error::new(ErrorKind::InvalidData, err))
+                    .map_err(::failure::err_msg)
```

Comment:
Setting the error kind to `InvalidData` wasn't used and was inconsistent with the rest of the code, so I guess it's ok to lose this bit of context here.